libusb/os/linux_usbfs.c

```diff
@@ -119,6 +119,8 @@ struct config_descriptor {
 struct linux_context_priv {
 	/* no enumeration or hot-plug detection */
 	int no_device_discovery;
+	/* no reap data after disconnection */
+	int no_reap_after_disconnect;
```

Comment:
Aren't these boolean?  (I realize the other was `int`, so you just probably copied it...)

Reply:
Yes, these are boolean, I don't know if I could include the "stdbool.h", so I follow the previous code.